x/feegrant/module/abci.go

```diff
@@ -1,14 +1,11 @@
 package module
 
 import (
-	"cosmossdk.io/x/feegrant/keeper"
+	"context"
 
-	sdk "github.com/cosmos/cosmos-sdk/types"
+	"cosmossdk.io/x/feegrant/keeper"
 )
 
-func EndBlocker(ctx sdk.Context, k keeper.Keeper) {
-	err := k.RemoveExpiredAllowances(ctx)
-	if err != nil {
-		panic(err)
-	}
+func EndBlocker(ctx context.Context, k keeper.Keeper) error {
+	return k.RemoveExpiredAllowances(ctx, 200)
```

Comment:
small comment on the picking of 200. could even say this was arbitrary.

Reply:
I'm going to put a comment saying it's arbitrary, in the meantime I was trying to make a proper benchmark using in-disk IAVL but I'm having some trouble

Reply:
we would need to test on a larger disk which we dont have the capability for yet.